x-pack/plugins/ui_actions_enhanced/public/drilldowns/components/drilldown_hello_bar/drilldown_hello_bar.tsx

```diff
@@ -31,31 +31,31 @@ export const DrilldownHelloBar: React.FC<DrilldownHelloBarProps> = ({
 }) => {
   return (
     <EuiCallOut data-test-subj={WELCOME_MESSAGE_TEST_SUBJ}>
-      <EuiFlexGroup>
-        <EuiFlexItem grow={false}>
-          <div style={{ marginLeft: '8px' }}>
+      <div style={{ marginBottom: -12 }}>
```

Comment:
`marginBottom: -12` - this doesn't do anything. there is !important in `eui` styles.
Not sure why changing this component was needed also

Reply:
With this change the vertical alignment between help icon and help text got worse: 

![94445467-c28b3000-01a7-11eb-9db0-d082b6ddfda8](https://user-images.githubusercontent.com/7784120/94450769-f2d5cd00-01ad-11eb-9905-be11313344c8.png)

Can we fix it?

Reply:
I tried a bit, but could not find a nice way, probably better ask EUI to fix it.

Reply:
If to pick between first and second image, I prefer 😅. 
Let's revert and create EUI issue? :D

Reply:
I think the second image where icon is aligned with the title and where top and bottom paddings are equal looks much better. Also I think the code of the second image is better because it gets rid of `style={{ marginLeft: '8px' }}`.

I suggest we keep the current change and leave it up to @elastic/eui-design if they want to move the icon lower or if they are OK with how it looks now.

Reply:
A heads up: will conflict https://github.com/elastic/kibana/pull/78801/files#diff-a247aa12e5b9e7ad760fe6c5880fc7efL36

Reply:
This is a completely custom implementation of the content for EuiCallout, so I'm not seeing what we can do on the EUI side. The ideal vertical alignment would be for the icon to either sit at the baseline of the text or centered with the first line of text not at the top of the letter. But the version without the bottom margin looks just fine to me.